include/drivers/st25r3911b.h

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef ST25R3911B_H_
+#define ST25R3911B_H_
+
+#include <zephyr/types.h>
+#include "st25r3911b_spi.h"
+
+/**
+ * @file
+ * @defgroup st25r3911b_common ST25R3911B NFC Reader common functionality
+ * @{
+ *
+ * @brief API for the ST25R3911B NFC Reader common functionality.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Maximum TX packet length. */
+#define ST25R3911B_MAX_TX_LEN 8191
+
+/** No automatic collision resolution. */
+#define ST25R3911B_NO_THRESHOLD_ANTICOLISION 0xFF
+
+/** Maximum collision resolution threshold. */
+#define ST25R3911B_MAX_THRESHOLD 0x0F
+
+/** Frame delay time (FDT) adjustment. */
+#define ST25R3911B_FDT_ADJUST 64
+
+/** Maximum FDT for the Mask Receive timer. */
+#define ST25R3911B_MASK_RECEIVER_MAX_FDT 0xFC0
+
+/** No-Response timer maximum value for step 64/fc. */
```

Comment:
Should this be "Non-response" or "No-response"? Google seems to prefer the latter. ;)

Reply:
"No-response" timer is a better name, also this is peripheral is called "No-re.." in chip spec.

Reply:
Great, that's what I used :)